assembl/static2/js/app/components/common/manageErrorAndLoading.jsx

```diff
@@ -19,24 +22,50 @@ type WrappedProps = {
   loading?: boolean
 };
 
+// Takes the type and return the right loader for error and loading
+const getLoaderToDisplay = (loaderType?: string, propsToPass: Props) => {
+  const type = loaderType || '';
+  switch (type) {
+  case 'watson': {
```

Comment:
Why having a switch/case for just 1 case ? Do we plan to have more ?

Reply:
@MilanBarande there two cases here. 'watson' and not 'watson' (default)